generator/shared_test.go

```diff
@@ -436,6 +436,10 @@ func TestShared_Issue1429(t *testing.T) {
 	log.SetOutput(ioutil.Discard)
 	defer log.SetOutput(os.Stdout)
 
+	if runtime.GOOS == "windows" {
+		t.Skip("on windows till the bug is resolved for ref resolution")
```

Comment:
ok this one is on my plate. I introduced a regression in go-openapi/analysis when running on windows.